cirq-core/cirq/ops/raw_types.py

```diff
@@ -399,11 +399,11 @@ def _qid_shape_(self) -> Tuple[int, ...]:
         """
 
     def _commutes_on_qids_(
-        self, qids: 'Sequence[cirq.Qid]', other: Any, atol: float
+        self, qids: 'Sequence[cirq.Qid]', other: Any, *, atol: float
```

Comment:
is there a reason not to set a default value here (and one directly below) as well?

Reply:
Changed.  I thought it might be too aggressive to put it in raw-types, but I think it may be good to have a global default, so I added it in on your suggestion.